arm-analysisservices/2016-05-16/swagger/analysisservices.json

```diff
@@ -410,7 +410,7 @@
       "description": "Properties of Analysis Services resource.",
       "type": "object",
       "properties": {
-        "provisioningState": {
+        "state": {
```

Comment:
@amarzavery @ravbhatnagar 

Here is the explanation from our team member:
I think it's clear for the reason why we need state. I would update 2nd iteration to surface both provisioningState and state, thanks 

We will use ‘State’ property to return custom terminal values for resource state.
ARM spec allows ‘Succeeded, Failed, Canceled’ values for resource terminal states.

We want to have custom values for our resources. E.g. AS server can  be in ‘Paused’ terminal state.
Response in that case is properties.State=’Paused’, properties.ProvisioningState=’Succeeded’.
Change to make State and ProvisioningState have different values will be made later.